PhysicsTools/NanoAOD/python/nanoDQM_cfi.py

```diff
@@ -809,5 +809,16 @@
                 Plot1D('rawMVAoldDMdR032017v2', 'rawMVAoldDMdR032017v2', 20, -1, 1, 'byIsolationMVArun2017v2DBoldDMdR0p3wLT raw output discriminator (2017v2)'),
             )
         ),
+	L1PreFiringWeight = cms.PSet(
+		sels = cms.PSet(),
+		plots = cms.VPSet(
+			Plot1D('L1PreFiringWeight_Nom', 'L1PreFiringWeight_Nom', 20, 0.8, 1.0, 'L1 prefiring weight nominal'),
```

Comment:
@perrotta Seems like using 1.0 as the range of the histogram here seems to move weights of 1 into the overflow?